daml/ContingentClaims/Inequality.daml

```diff
@@ -0,0 +1,21 @@
+module ContingentClaims.Inequality where
+
+import ContingentClaims.Observation (Observation, eval)
+
+-- | This is either `time ≥ t | f t x ≤ f t x`
+data Inequality t x o = TimeGte t | Lte (Observation t x o, Observation t x o)
```

Comment:
Do you think these should go in `Internal` too? i.e. just use `at` or `>=`?

Reply:
I personally like to use `TimeGte` and `(≤)` as constructors, as I find them the most descriptive.